plugins/dra-ocp-validator/commands/validate.md

```
@@ -0,0 +1,3 @@
+=== File too long to show. Ends with: ===
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)